bso/structural_design/element/quad_hexahedron.cpp

```diff
@@ -198,6 +209,36 @@ namespace bso { namespace structural_design { namespace element {
 		
 	} // dtor
 
+	void quad_hexahedron::computeResponse(load_case lc)
+	{ //
+		Eigen::VectorXd elementDisplacements(mSM.rows());
+		elementDisplacements.setZero();
+		auto dispIte = elementDisplacements.data();
+
+		for (const auto& i : mNodes)
+		{
+			for (unsigned int j = 0; j < 6; ++j)
+			{
+				if (mEFS(j) == 1)
+				{
+					*dispIte = i->getDisplacements(lc)(j);
+					++dispIte;
+				}
+			}
+		}
+		mDisplacements[lc] = elementDisplacements;
+		mEnergies[lc] = 0.5 * elementDisplacements.transpose() * mSM * elementDisplacements;
+		mTotalEnergy += mEnergies[lc];
```

Comment:
to avoid duplicate code, please replace call the code of the parent class here by replacing this block of code with `element::compteResponse(lc)`